### ISSUE TITLE
Add missing `platform_aws_t4g`

### ISSUE THREAD

**Issue by davidkopp** [Boavizta/boaviztapi#348]
## Bug description

Somehow the data for the cloud platform `platform_aws_t4g` is missing. For example, no data is returned for the aws instance type `t4g.large`. The data was available in the past.

## To Reproduce

```
curl -X 'POST' 'https://api.boavizta.org/v1/cloud/instance?verbose=false&criteria=gwp' -H 'accept: application/json' -H 'Content-Type: application/json' -d '{
  "provider": "aws",
  "instance_type": "t4g.large"
}'
```

## JSON OUTPUT

```json
{
  "detail": "Cloud platform platform_aws_t4g not found. Please add it to the server archetypes. For more information, please check the documentation https://doc.api.boavizta.org/contributing/server/."
}
```

## Additional context

Missing data should be added to https://github.com/Boavizta/boaviztapi/blob/main/boaviztapi/data/archetypes/server.csv

**Comment by da-ekchajzer** [comment]
Hey, i am on it.
for your information, if you were using the API in production, you would have to host it yourself so as not to be dependent on version upgrades.

**Comment by davidkopp** [comment]
Thanks :) 
I think the following line is missing in the file `server.csv`:

```csv
platform_aws_t4g,AWS,rack,1,,,Annapurna Labs Graviton2,,8,32,0,0,0,0,0,,0,2;2;2,2.99;1;5,50;0;100,1,35040,0.33;0.2;0.6,RAM configuration was not verified
```

**Comment by da-ekchajzer** [comment]
Yes thank you ! I am adding a new test to test if all platforms used by instances exist so that it doesn't happen again.

**Comment by da-ekchajzer** [comment]
Should be fixed on dev (dev.api.boavizta.org) could you try ?

**Comment by davidkopp** [comment]
Looks good to me! Thanks 😀

**Comment by da-ekchajzer** [comment]
Nice it will be available in v1.3.4. I am releasing right now
